framework/test/mocha/unit/components/storage/entities/account.js

```diff
@@ -628,6 +630,112 @@ describe('Account', () => {
 		});
 
 		describe('filters', () => {
+			describe('asset_contains', () => {
+				let aliceAccount = null;
+				let bobAccount = null;
+
+				beforeEach(async () => {
+					aliceAccount = new accountFixtures.Account({
+						asset: {
+							name: 'Alice',
+							company: 'Acme Corp',
+						},
+					});
+					bobAccount = new accountFixtures.Account({
+						asset: {
+							name: 'Bob',
+							company: 'Acme Corp',
+							tags: ['blockchain', 'crypto'],
+						},
+					});
+
+					await AccountEntity.create([aliceAccount, bobAccount]);
+				});
+
+				it('should return single result when asset value contains unique property at the top level', async () => {
+					const filters = { asset_contains: { name: 'Alice' } };
+					const accounts = await AccountEntity.get(filters);
+
+					expect(accounts.length).to.be.eql(1);
+					expect(accounts[0].address).to.be.eql(aliceAccount.address);
+				});
+
+				it('should return result when asset value contains given array element', async () => {
+					const filters = { asset_contains: { tags: ['crypto'] } };
```

Comment:
Sorry, I want to have more tests 🗡😀 
Can you cover these scenarios:
- Match an array with multiple correct elements: `{ asset_contains: { tags: ['crypto', 'blockchain'] } };`
- See what happens for an empty array `{ asset_contains: { tags: [] } };`
- See what happens for an empty string `{ asset_contains: { name: '' } };`
- See what happens in case you filter for a wrong property that doesn't exist: `{ asset_contains: { not-name: 'Bob' } };